insights/combiners/satellite_version.py

```diff
@@ -65,6 +65,7 @@
 from insights.core.plugins import combiner
 from insights.parsers.satellite_version import Satellite6Version as Sat6Ver
 from insights.parsers.installed_rpms import InstalledRpms
+from insights.parsers import SkipException
```

Comment:
Instead of using parser exceptions in combiners, let's move toward using `insights.SkipComponent`. I have a [PR](#755) open to support it.